src/zebras/_freq.py

```diff
@@ -126,21 +136,39 @@ def __get_validators__(cls):
         yield freq
 
     @classmethod
```

Comment:
Let's not make this a classmethod, I think this can be a plain function.

Reply:
I'll move it out of `Freq` class.